bftengine/src/bftengine/CollectorOfThresholdSignatures.hpp

```diff
@@ -182,7 +182,6 @@ class CollectorOfThresholdSignatures {
   void initExpected(SeqNum seqNumber, ViewNum view, Digest& digest) {
     ConcordAssert(seqNumber != 0);
     ConcordAssert(expectedSeqNumber == 0);
-    ConcordAssert(!processingSignaturesInTheBackground);
```

Comment:
why it's not valid anymore?

Reply:
(a) If we directly put the signature/digest we should be caring if there is another processing in the background.
(b) Specifically, with a single replica cluster, when it recover, once it pushes the paritailCommit message (of itself) it is already have enough data to start and process the multi signature, however, it is not needed because immidetly after it directly put the fullCommit message signature